decompiler/frontend/binaryninja/handlers/globals.py

```diff
@@ -25,6 +25,7 @@ def lift_global_variable(self, variable: DataVariable, **kwargs) -> UnaryOperati
         parent_addr = kwargs["parent_addr"]
         addr = variable.address
 
+        # TODO: why do we need that? variable.name should work, ImportAddressSymbols are now lifted as &global
```

Comment:
It is not a leftover, I don't like the function like it is. So I was planning to create follow up issue to refactor `constants.py` and `globasl.py` so that they are easier to comprehend.

Reply:
Alright - I agree with you here. Just link the issue in this thread and remove the comment for now

Reply:
#110  is created. Tried to keep issue description as generic as possible. Let me know if more detailed description is needed.